src/ppk2_api/ppk2_api.py

```diff
@@ -338,28 +355,35 @@ def get_samples(self, buf):
         sample_size = 4  # one analog value is 4 bytes in size
         offset = self.remainder["len"]
         samples = []
+        raw_digital_output = []
 
         first_reading = (
             self.remainder["sequence"] + buf[0:sample_size-offset])[:4]
         adc_val = self._digital_to_analog(first_reading)
-        measurement = self._handle_raw_data(adc_val)
+        measurement, bits = self._handle_raw_data(adc_val)
         if measurement is not None:
             samples.append(measurement)
+        if bits is not None:
+            raw_digital_output.append(bits)
 
         offset = sample_size - offset
 
         while offset <= len(buf) - sample_size:
             next_val = buf[offset:offset + sample_size]
             offset += sample_size
             adc_val = self._digital_to_analog(next_val)
-            measurement = self._handle_raw_data(adc_val)
+            measurement, bits = self._handle_raw_data(adc_val)
             if measurement is not None:
                 samples.append(measurement)
+            if bits is not None:
+                raw_digital_output.append(bits)
 
         self.remainder["sequence"] = buf[offset:len(buf)]
         self.remainder["len"] = len(buf)-offset
 
-        return samples  # return list of samples, handle those lists in PPK2 API wrapper
+        # return list of samples and raw digital outputs
+        # handle those lists in PPK2 API wrapper
+        return samples, raw_digital_output  
```

Comment:
FYI this is a breaking change for any code that uses `get_samples`.

Reply:
The example was updated to show that :) Update your code with `get_samples, _ = ppk2.get_samples(data)`

Reply:
Yeah it all made sense once I found this PR! I had a tester rig break on the update from 0.9.1 and 0.9.2. I'll pin the exact version next time!